zeppelin-web/src/app/notebook/paragraph/paragraph.css

```diff
@@ -382,8 +382,9 @@ table.table-shortcut {
 
 /** set cursor color */
 #main .emacs-mode .ace_cursor {
-  background: #C0C0C0!important;
+  background: #000 !important;
   border: none !important;
+  width: 1px !important;
```

Comment:
could we canhge make width be thicker? This is the default version in ace editor

- https://ace.c9.io/

![image](https://user-images.githubusercontent.com/4968473/28761777-104fd7f0-75ec-11e7-9279-7099bf089b7d.png)